src/ui/CookiesPopup.svelte

```diff
@@ -29,20 +30,33 @@
     isVisible = false
     showManageCookiesDialog()
   }
+
+  function onCloseClick() {
+    isVisible = false
+  }
 </script>
 
 {#if isVisible}
   <div class="cookies border box {className} {style}">
+    {#if $IsMobile}
+      <button class="btn close" on:click={onCloseClick}>
+        <Svg id="close" w="12" />
+      </button>
+    {/if}
     <Svg illus id="cookies" class="$style.pic" />
-    <h2 class="body-2 txt-m mrg-s mrg--b">We are using cookies to improve your experience!</h2>
-    <h4 class="body-3 mrg-xl mrg--b c-waterloo">
+    <h2 class="{$IsMobile ? 'h4 txt-center mrg-l' : 'body-2 mrg-s'} txt-m mrg--b">
+      We are using cookies to improve your experience!
+    </h2>
+    <h4 class="{$IsMobile ? 'body-2' : 'body-3'} mrg-xl mrg--b c-waterloo">
       By clicking “Allow all”, you agree to use of all cookies. Visit our
       <a href="https://santiment.net/cookies/" target="_blank" class="link-pointer"
         >Cookies Policy</a
       > to learn more.
     </h4>
-    <button class="btn-1" on:click={onAllowAllClick}>Allow all</button>
-    <button class="manage btn-2 mrg-m mrg--l" on:click={onManageClick}>Manage cookies</button>
+    <button class="btn-1 {$IsMobile ? 'body-2' : ''}" on:click={onAllowAllClick}>Allow all</button>
+    <button class="manage btn-2 {$IsMobile ? 'body-2' : ''} mrg-m mrg--l" on:click={onManageClick}
```

Comment:
Add `class:body-2={$IsMobile}` to `.cookies` container. Everything else will be `body-3` by default